lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-your-own.adoc

```diff
@@ -0,0 +1,44 @@
+= Building Linear Models
+++++
```

Comment:
noting that the title of this page and the file have diverged significantly

Reply:
Fixed.